tools/dependency-check/known-dependencies.txt

```diff
@@ -1,4 +1,3 @@
-FastInfoset-1.2.15.jar
 ST4-4.3.4.jar
 accessors-smart-2.4.7.jar
```

Comment:
According to https://github.com/apache/eventmesh/pull/4863#issuecomment-2076542000, shall we remove `ST4` together in this PR?

Reply:
After rebasing the code, I used a plugin to check and found no errors. The status is passed

![image](https://github.com/apache/eventmesh/assets/15797831/621b3e57-e920-4c10-96b5-963af9d0cb9d)

Reply:
The license content of https://www.antlr.org/license.html is identical with BSD-3-Clause. It seems to be an error of upstream supply chain, and `ST4` is compatible.